liquibase-core/src/main/java/liquibase/changelog/ChangeLogIterator.java

```diff
@@ -179,7 +179,7 @@ protected String createKey(ChangeSet changeSet) {
         Labels labels = changeSet.getLabels();
         ContextExpression contexts = changeSet.getContextFilter();
 
-        return changeSet.toString(true)
+        return changeSet.toString(false)
```

Comment:
This class is missing JavaDoc annotations. Should I raise a separate GH Issue to address this?